Create React App block: Fix JS and JSX paths (#2074)
ezsystems/developer-documentation: docs/content_management/pages/react_app_block.md

Each configured React app block has an identifier and the following settings:

| Setting      | Description                                                                                                                                                  |
|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `name`       | Name of the block used in the Page Builder interface. (Used as Component name if no `component` is provided.)                                                |
| `category`   | Category in the Page Builder elements menu that the block is shown in.                                                                                       |
| `thumbnail`  | Thumbnail used in the Page Builder elements menu.                                                                                                            |
| `visible`    | (Optional) Toggles the block's visibility in the Page Builder elements menu. Remove the block from the layout before you publish another version of the page.|

> | `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` is used as Component name.)             |

Reviewer: ```suggestion
| `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` is used as a Component name.)             |
```

Author: @MagdalenaZuba In this case, I would use `the` as it must match the uniq Component name from conf file.
```suggestion
| `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` is used as the Component name.)             |
```

> | `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` will be used as Component name.)             |

Author: ```suggestion
| `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` is used as Component name.)             |
```

> | `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` will be used as Component name.)             |

Reviewer: ```suggestion
| `component`  | (Optional) React App Component name used in `assets/page-builder/react/blocks` directory. (If not given, `name` is used as a Component name.)             |
```
